### DISCUSSION TITLE
How to customize navigation bars for internationalization (i18n)?

### DISCUSSION THREAD

**Discussion by rockbenben** [facebook/docusaurus#8844] [Q&A] [answered]
I am planning to include operational documentation in the navigation bar, but it will link to different URLs. Is it possible to customize the navigation bar for each language?

I have attempted to set variable functions within "docusaurus.config.js" and also tried setting different components, but both methods have failed.
```
my-docusaurus-project/
├── docusaurus.config.js
└── src/
    └── components/
        ├── LocalizedNavbar/
        │   ├── Navbar.en.js
        │   └── Navbar.zh.js
        └── LocalizedNavbar.js
```

**Comment by slorber** [comment, marked as answer]
I don't really understand what you are trying to do.

Maybe the v2.4 DOCUSAURUS_CURRENT_LOCALE  env variable could help define different navbar items for each locale? 
https://github.com/facebook/docusaurus/pull/8677

**Reply by rockbenben** [reply]
> I don't really understand what you are trying to do.
> 
> Maybe the v2.4 DOCUSAURUS_CURRENT_LOCALE env variable could help define different navbar items for each locale? #8677

Thanks, that is splendid. The navigation bar in my project currently directs only to the default language page, whereas the navigation bar for other languages should direct to alternative links or be concealed. The use of DOCUSAURUS_CURRENT_LOCALE can effectively resolve this quandary.

![Chinese version](https://user-images.githubusercontent.com/28252913/230182000-c7032c81-e767-4128-8c3e-2a0ccb09b8d0.png)
![English Version](https://user-images.githubusercontent.com/28252913/230182124-309e1dfc-cf67-4b8d-a787-a0ddc883ed3b.png)
